examples/stories/keyframes-tagged-template-expression.tsx

```diff
@@ -3,7 +3,6 @@ import { css, keyframes, styled } from '@compiled/react';
 import '../keyframes/globals';
 import { fadeOut as shadowedFadeOut } from '../keyframes/tagged-template-literal';
 
-// TODO Add css prop using a call expression mixin once supported: https://github.com/atlassian-labs/compiled/issues/789
```

Comment:
Just FYI, this comment was for updating any `css={{}}` usages to `css={css({})}` but it's not a big deal